sinonjs/sinon
# Fix 2377: Throw error when trying to stub non-configurable or non-writable properties

#### Purpose (TL;DR) - mandatory

Fixes issue https://github.com/sinonjs/sinon/issues/2377

#### How to verify - mandatory

1. Check out this branch
2. `npm install`
3. Check out the tests I created. That would probably be the easiest way to verify

#### Checklist for author
